independent-publisher-connectors/PaySpace/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4476 @@
+                  "description": "Classification"
+                },
+                "AttachmentDescription": {
+                  "type": "string",
+                  "description": "AttachmentDescription"
+                },
+                "AttachmentName": {
+                  "type": "string",
+                  "description": "AttachmentName"
+                },
+                "AttachmentUrl": {
+                  "type": "string",
+                  "description": "AttachmentUrl"
+                },
+                "Attachment": {
+                  "type": "string",
+                  "description": "Attachment"
+                },
+                "CustomFields": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "Code": {
+                        "type": "string",
+                        "description": "Code"
+                      },
+                      "Label": {
+                        "type": "string",
+                        "description": "Label"
+                      },
+                      "Value": {
+                        "type": "string",
+                        "description": "Value"
+                      }
+                    }
+                  },
+                  "description": "CustomFields"
+                }
+              }
+            }
+          }
+        }
+      },
+      "delete": {
+        "summary": "Delete a single employee attachment record",
+        "description": "Delete a single employee attachment record based on the specified AttachmentId.",
+        "operationId": "DeleteASingleEmployeeAttachmentRecord",
+        "parameters": [
+          {
+            "name": "company-id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "{{company-id}} The Company Id",
+            "x-ms-summary": "{{company-id}}"
+          },
+          {
+            "name": "AttachmentId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "{{AttachmentId}} The AttachmentId",
+            "x-ms-summary": "AttachmentId"
+          },
+          {
+            "name": "CustomAuthHeader",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Bearer token value (without the keyword Bearer)",
+            "x-ms-summary": "Bearer token"
+          },
+          {
+            "name": "CustomEnvironmentHeader",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Variable that is either api (for PROD) or apistaging (for STAGING)",
+            "x-ms-summary": "Environment Variable"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@mint-virgil empty schema is not allowed. Please remove or define a proper schema.
Please fix all empty schemas in your connector swagger file.

Reply:
updated in latest commit